src/Mod/Fem/App/FemPostPipeline.h

```diff
@@ -23,30 +23,65 @@
 #ifndef Fem_FemPostPipeline_H
 #define Fem_FemPostPipeline_H
 
+#include "Base/Unit.h"
+#include "FemPostGroupExtension.h"
+
+#include "FemPostFilter.h"
 #include "FemPostFunction.h"
 #include "FemPostObject.h"
 #include "FemResultObject.h"
 
 #include <vtkSmartPointer.h>
+#include <vtkUnstructuredGridAlgorithm.h>
+#include <vtkInformation.h>
+#include <vtkInformationVector.h>
```

Comment:
Can we forward declare this?

Reply:
We can for vtkInformation and vtkInformationVector. Will be added.